lib/Net/Async/Blockchain/Omni.pm

```diff
@@ -125,23 +142,63 @@ async sub transform_transaction {
 
     return undef unless $transaction_type;
 
-    my $transaction = Net::Async::Blockchain::Transaction->new(
-        currency     => $self->currency_symbol,
-        hash         => $decoded_raw_transaction->{txid},
-        block        => $block,
-        from         => $from->{address},
-        to           => $to->{address},
-        amount       => $amount,
-        fee          => $fee,
-        fee_currency => FEE_CURRENCY,
-        type         => $transaction_type,
-        property_id  => $received_transaction->{propertyid},
-        timestamp    => $received_transaction->{blocktime},
-    );
+    if ($omni_transaction->{type} eq "Send All") {
+
+        for my $data ($omni_transaction->{subsends}->@*) {
+            $sendall{$data->{propertyid}} = $data->{amount};
+        }
+
+        for my $propertyid (keys %sendall) {
+
+            $transaction = Net::Async::Blockchain::Transaction->new(
```

Comment:
here we assign all transactions to one variable, after this loop, we will have only last value to return. Is it expected behavior?

Reply:
Updated the transaction 👍